inc/thorvg.h

```diff
@@ -1349,21 +1359,46 @@ class TVG_EXPORT Saver
     ~Saver();
 
     /**
-     * @brief Saves all the paints from the tree in a binary format.
+     * @brief Export the given @p paint data to the given @p path
      *
-     * @param[in] paint The root paint to be saved with all its nodes.
-     * @param[in] path A path to the file, in which the data is to be saved.
+     * @param[in] paint The paint to be saved with all its associated properties.
+     * @param[in] path A path to the file, in which the paint data is to be saved.
      *
      * @retval Result::Success When succeed.
-     * @retval Result::InvalidArguments the @p path is invalid.
-     * @retval Result::FailedAllocation An internal error with a memory allocation for the Saver object.
-     * @retval Result::MemoryCorruption When casting in the internal function implementation failed.
+     * @retval Result::NonSupport When trying to save a file with an unknown extension nor non supported format.
      * @retval Result::Unknown Others.
      *
+     * @note Saving can be asynchronous if the assigned thread number is greater than zero. To guarantee the saving is done, call sync() afterwards.
+     * @see Saver::sync()
+     *
+     * @BETA_API
      */
     Result save(std::unique_ptr<Paint> paint, const std::string& path) noexcept;
+
+    /**
+     * @brief Guarantees that the saving task is finished.
+     *
+     * The behavior of the saver will work on sync/async based on the threading setting of the initializer.
```

Comment:
The behavior of the saver will work on a sync/async basis, depending on the threading setting of the Initializer.